### DISCUSSION TITLE
Grain boundaries with different misorientation

### DISCUSSION THREAD

**Discussion by alielasheri** [mtex-toolbox/mtex#585] [Ask Anything] [answered]
Hello, 

I would like to have an answer for how can I visualize Low, medium and high angle boundaries. let's consider the thresholds are 
5 deg, 10 deg and 15 deg.

I tried to apply what the method in this topic https://mtex-toolbox.github.io/SubGrainBoundaries.html. but it I couldn't reach the desired results 

Many thanks in advance

**Comment by kilir** [comment, marked as answer]
Hi,
have a look here: https://mtex-toolbox.github.io/BoundaryProperties.html 
Simple example:
```matlab
mtexdata titanium
grains  =ebsd.calcGrains;
gbt = grains.boundary('t','t');

cond = gbt.misorientation.angle > 20*degree & ...
           gbt.misorientation.angle < 40*degree;

plot(gbt(~cond),'linecolor','k')
hold on
plot(gbt(cond),'linecolor','r','DisplayName','boundaries in range 20-40 degree')
hold off
legend
```
Cheers,
Rüdiger
